lib/gemstash/gem_fetcher.rb

```diff
@@ -8,7 +8,7 @@ module Gemstash
   class GemFetcher
     def initialize(http_client)
       @http_client = http_client
-      @valid_headers = Set.new(["etag", "content-type", "content-length", "last-modified"])
+      @valid_headers = Set.new(%w[etag content-type content-length last-modified])
```

Comment:
Why is this needed?

Reply:
I reverted the change since it was related to a rubocop warning.